### DISCUSSION TITLE
Apicurio Integration Microcks and Mock Url

### DISCUSSION THREAD

**Discussion by aaniceto** [microcks/microcks#743] [Q&A] [answered]
Hi all,

This should be basic, but I've looked around as much as possible and used egrep everywhere.

While doing the integratin between Apicurio and Microcks (which is fully working), the link from Apicurio points to Microcks API (port 8080 in this case) instead of 4200.

And on the Microcks UI the url/port for the Mock URL points to 4200 instead of 8080.. Clearly somewhere this is mixed.. Any ideas?

Thanks,
Alex

**Comment by lbroudoux** [comment, marked as answer]
Hi there,

You're right: Microcks is using 2 ports when running in development mode (`4200` for UI and `8080` for server). Apicurio Studio is intended to be connected to a Microcks instance running in prod mode that's using just 1 port - typically `8080` (UI part is embedded in server in that mode).

Best,

**Reply by aaniceto** [reply]
Hi,

Thanks. That makes sense. I was following the manual on webapp/README.md - that now I see it states exactly as you said.

Anyway we would like to run this withour docker, kuber, etc. "manually" if you want to call it this way.

Is there a way to run in ? Following the CONTRIBUTING.md file

$ cd webapp
$ mvn -Pprod package
$ java -jar target/microcks-app-1.6.2-SNAPSHOT.jar

This will run microcks but just the server part again. At least I was not able to access the UI.

Thanks!

**Reply by aaniceto** [reply]
All working!!!! Thanks.
